README.md

```diff
@@ -96,6 +96,7 @@ zulip.callEndpoint('/messages', 'POST', params);
 | `zulip.typing.send()` | POST `/typing` | can be used to send a typing notification. The parameters required are `to` (either a username or a list of usernames) and `op` (either `start` or `stop`). |
 | `zulip.users.retrieve()` | GET `/users` | retrieves all users for this realm. |
 | `zulip.users.me.pointer.retrieve()` | GET `/users/me/pointer` | retrieves a pointer for a user. The pointer is the id of the last message the user read. This can then be used as an anchor message id for subsequent API calls. |
+| `zulip.users.me.getProfile.retrieve()` | GET `/users/me/getProfile` | retrieves the profile of the user/bot. |
```

Comment:
Sorry for bugging you for minor changes. This one's left. :)

Reply:
@aero31aero  Really sorry for that! Hope everything is fine now.